Windows error

Hi there ,
Trying to get the package working but getting the following error in Windows

```
module.js:474
    throw err;
    ^

Error: Cannot find module '.\pre-commit-utils\git-manager'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\dev\bd-events\.git\hooks\pre-commit.js:9:20)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
```

Any ideas? 
Have 
`"precommit": "npm test",` in my package.json file

--- a/scripts/pre-commit.js
+++ b/scripts/pre-commit.js
@@ -6,9 +6,9 @@ const path = require('path');
 require('shelljs/global');
 
 const relativePrecommitUtilsFolder = "pre-commit-utils";
-const GitManager = require(`.${path.sep}${relativePrecommitUtilsFolder}${path.sep}git-manager`);
-const PackageManagerUtils = require(`.${path.sep}${relativePrecommitUtilsFolder}${path.sep}package-manager-utils`);
-const gulpUtils = require(`.${path.sep}${relativePrecommitUtilsFolder}${path.sep}gulp-utils`);
+const GitManager = require(`./${relativePrecommitUtilsFolder}/git-manager`);
+const PackageManagerUtils = require(`./${relativePrecommitUtilsFolder}/package-manager-utils`);
+const gulpUtils = require(`./${relativePrecommitUtilsFolder}/gulp-utils`);
 
 // Why do we use spawn and not a regular shelljs?
 // We want to preserve the colors of the output, so until the feature will be implemented

